### PR TITLE
Update initial setup steps and sample config

### PR DESCRIPTION
**Pull request by ninjacatt**
Fixes initial setup documents (Sorry there isn't bug tracking this yet)
While I were going over the readme.md to setup HR for the first time, I noticed some small information that missing and mismatch. So propose this update to help others if possible.

**Changes proposed in this pull request:**
- Update readme to include install using npm and bower
- Update sample config to use the hradmin account which is being used in default script to create couchdb admin and in the readme.

cc @HospitalRun/core-maintainers

### CODE REVIEW

**Comment by BogdanAlexandru** [comment]
Note that `npm install` and `bower install` are part of `script/bootstrap` which is executed in the installation steps.

**Comment by ninjacatt** [comment]
Ah good point. I went over the server script but completely missed the bootstrap one. I could put a note to say "make sure have those installed first" or just recall this PR all together :)
		

		
	





On Sun, Mar 26, 2017 at 1:46 PM -0700, "Bogdan Alexandru Marginean" <notifications@github.com> wrote:












Note that npm install and bower install are part of script/bootstrap which is executed in the installation steps.

File: server/config-example.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ var config = {
   couchDbPort: '5984',
   couchDbUseSsl: false,
   couchDbChangesSince: 'now',
-  couchAdminUser: 'couchadmin',
+  couchAdminUser: 'hradmin',
```
Comment (jkleinsc) [inline server/config-example.js]: This is intentionally couchadmin, not hradmin.

There are two users that the script sets up:
1) _couchadmin_ - CouchDB admin.  This is a [CouchDB server administrator](http://docs.couchdb.org/en/2.0.0/config/auth.html?highlight=Admin%20party#server-administrators) which is a CouchDB super user.  This is used by hospitalrun-server-routes to do some user setup that a regular user can't do.
2) _hradmin_ - This is the default user to login to HospitalRun with.  This user is not a CouchDB server admin.

**Comment by jkleinsc** [comment]
I am going to close this PR.   @ninjacatt if you want to update the readme to reference script/bootstrap in the start section, feel free to open a new PR with that change.

File: server/config-example.js (hunk 1 above)
Comment (sukima) [inline server/config-example.js]: Can this be added to the README. The difference between `hradmin` and `couchadmin` really confused me (I didn't even know there was a `couchadmin`) till I saw this comment.

**Pull request closed without merging**